examples/layers/preprocessing/random_flip_demo.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+   random_flip_demo.py shows how to use the RandomFlip preprocessing layer
+   for object detection. An image is downloaded from URL, both the image and
```

Comment:
lets actually follow the example of https://github.com/keras-team/keras-cv/pull/373 for the demos